containers/src/Kokkos_DynamicView.hpp

```diff
@@ -299,19 +299,21 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
 
     // *m_chunks[m_chunk_max] stores the current number of chunks being used
     uintptr_t* const pc = reinterpret_cast<uintptr_t*>(m_chunks + m_chunk_max);
-
+    std::string label =
```

Comment:
GCC-4.8.4 emits a `-Wshadow` warning because of
https://github.com/kokkos/kokkos/blob/b7d68ced88e5e02897a191abadcded8ae765e92e/containers/src/Kokkos_DynamicView.hpp#L214

Reply:
ok fixed it